jenkins-build.sh

```diff
@@ -142,6 +182,21 @@ else
                               ldap_admin_pwd=$1;
                           fi
                           ;;
+       --generate_support_bundle ) shift;
+                                   if [ ! -z "$1" ]; then
+                                      generate_support_bundle=$1;
+                                   fi
+                                   ;;
+       --job_id ) shift;
+                  if [ ! -z "$1" ]; then
```

Comment:
pls check if jenkins-job script can generate job id before running the actual jenkins job

Reply:
Yes, Talked to Shailesh, he said we can get the job id. But I will confirm once again

Reply:
Confirmed again with Shailesh. His reply on Team's chat:

[Yesterday 7:05 PM] Shailesh Vaidya
    Hi Amit - Sorry caught up in couple of things so could not reply. Yes we have job-id available an env variable during Jenkins job execution.